tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -609,6 +627,18 @@ platform_tests/test_reboot.py:
     conditions:
       - "hwsku in ['Celestica-E1031-T48S4']"
 
+platform_tests/test_reboot.py::test_power_off_reboot:
+  skip:
+    reason: "Skip power off reboot test for Wistron"
+    conditions:
+      - "'sw_to3200k' in hwsku"
```

Comment:
The conditional mark is using "longest match" algorithm.

When we run test case `platform_tests/test_reboot.py::test_power_off_reboot`, the longest match would be this entry. The conditions defined in `platform_tests/test_reboot.py` will not be evaluated. This means that this test case will be executed on platform `Celestica-E1031-T48S4`. This is unexpected.

Need to change the condition string from 
```
      - "'sw_to3200k' in hwsku"
```
to
```
      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku"
```